packages/core/src/api/channel.ts

```diff
@@ -21,6 +21,7 @@ import {
 	type RESTPostAPIChannelMessageCrosspostResult,
 	type RESTPostAPIChannelMessageJSONBody,
 	type RESTPostAPIChannelMessageResult,
+	type RESTGetAPIChannelWebhooksResult,
```

Comment:
This import is not sorted, it should be on line 15 (`RESTGetAPIChannelUsersThreadsArchivedResult` → `RESTGetAPIChannelWebhooksResult` → `RESTPatchAPIChannelJSONBody`).